schema.xsd

```diff
@@ -236,35 +270,57 @@
             <xs:element name="items">
                 <xs:complexType>
                     <xs:sequence>
-                        <xs:element name="item">
+                        <xs:element name="item" minOccurs="0" maxOccurs="unbounded">
                             <xs:complexType>
                                 <xs:sequence>
                                     <xs:element name="name" type="xs:string"/>
                                     <xs:element name="weight" type="xs:decimal"/>
 
-                                    <!-- Range filters only: -->
-                                    <xs:element name="parameters" type="range"/>
+                                    <!-- For filters other than range filters. -->
+                                    <xs:element name="frequency" type="xs:nonNegativeInteger" minOccurs="0" maxOccurs="1" />
```

Comment:
`1` is the default value of `maxOccurs`, so it shouldn't be necessary to specify it explicitly.